src/pynwb/base.py

```diff
@@ -50,26 +50,25 @@ def add_container(self, **kwargs):
         '''
         Add an NWBContainer to this ProcessingModule
         '''
-        warn(PendingDeprecationWarning('add_container will be replaced by add'))
+        warn("add_container is deprecated. Use add instead.", DeprecationWarning)
```

Comment:
```suggestion
        warn("add_container is deprecated and will be removed in PyNWB 4.0. Use add instead.", DeprecationWarning)
```

Reply:
@stephprince Can you update the other messages too to say x will be removed in PyNWB 4.0?